packages/grid/_modules_/grid/hooks/root/useGridContainerProps.ts

```diff
@@ -14,46 +15,90 @@ import { useGridSelector } from '../features/core/useGridSelector';
 import { useGridState } from '../features/core/useGridState';
 import { gridDensityRowHeightSelector } from '../features/density/densitySelector';
 import { visibleGridRowCountSelector } from '../features/filter/gridFilterSelector';
-import { gridPaginationSelector } from '../features/pagination/gridPaginationSelector';
+import {
+  gridPageSizeSelector,
+  gridPageSelector,
+} from '../features/pagination/gridPaginationSelector';
 import { optionsSelector } from '../utils/optionsSelector';
 import { useLogger } from '../utils/useLogger';
 import { useGridApiEventHandler } from './useGridApiEventHandler';
-
-export const useGridContainerProps = (apiRef: GridApiRef) => {
+import { GridComponentProps } from '../../GridComponentProps';
+
+function getBrowserScrollbarSize(doc: Document, element: HTMLElement): number {
```

Comment:
Should we move this to the `utils.ts` file?

Reply:
I was in a dedicated hook before but I had to move it closer to where it's used.
But I agree, it would be better in `utils.ts`